### ISSUE TITLE
transition options should be optional

### ISSUE THREAD

**Issue by ramiroaisen** [halfnelson/svelte-type-checker#2]
Transition options should be optional
They are required right now

**Comment by halfnelson** [comment]
I agree, Might need a PR to svelte since it is just using their definitions for transition (from memory)

**Comment by ramiroaisen** [comment]
> I agree, Might need a PR to svelte since it is just using their definitions for transition (from memory)

You can use the definitions in https://github.com/pyoner/svelte-types/blob/master/types/svelte/transition.d.ts
